src/Compiler/TypedTree/TcGlobals.fs

```diff
@@ -184,6 +184,8 @@ let tname_IAsyncResult = "System.IAsyncResult"
 [<Literal>]
 let tname_IsByRefLikeAttribute = "System.Runtime.CompilerServices.IsByRefLikeAttribute"
 
+
```

Comment:
Line 1747, is member val isSystem_Runtime_CompilerServices_IsReadOnlyAttributeAvailable still needed for anything?
If yes, is it still needed as a public value of TcGlobals?

(I would like to see it removed from here to prevent incorrect usage)

Reply:
@T-Gro --- I have yet to figure out what the actual abstraction will be for IsReadOnlyAttributeAvailable on tcGlobals.